bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/model/BusinessPartnerSaaSWithBpn.kt

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright (c) 2021,2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+package org.eclipse.tractusx.bpdm.gate.model
+
+import org.eclipse.tractusx.bpdm.common.dto.saas.BusinessPartnerSaas
+
+data class BusinessPartnerSaaSWithBpn(
```

Comment:
Minor: To be consistent with the other symbols I would rather spell it `Saas` instead of `SaaS`.
This is a matter of taste but I would write a whole abbreviation, whether CDQ or SaaS, as one word in camelcase, e.g. "PartnerCdqDto", not "PartnerCDQDto".

Reply:
fixed

Reply:
agree here, done